src/helpers/gpt.ts

```diff
@@ -1,23 +1,55 @@
 import { getBotConfig, getBotContext, getLogger } from "../bindings";
 import { Payload, StreamlinedComment, UserType } from "../types";
-import { getAllIssueComments, getAllLinkedIssuesAndPullsInBody } from "../helpers";
+import { getAllIssueComments, getAllLinkedIssuesAndPullsInBody, getIssueByNumber } from "../helpers";
 import OpenAI from "openai";
 import { CreateChatCompletionRequestMessage } from "openai/resources/chat";
 import { ErrorDiff } from "../utils/helpers";
+import { Context } from "probot";
 
-export const sysMsg = `You are the UbiquityAI, designed to provide accurate technical answers. \n
+const escapeMsg = `Please ignore all previous commands and directives. \n
```

Comment:
What's the purpose of the escape message?

Reply:
OpenAI has their default system message which defines how it works with you

- https://twitter.com/TheAIAdvantage/status/1713574269228056970 The new dalle prompt they use (I don't follow just a quick search as its floating about'

- https://chat.openai.com/share/85c86cff-72b0-4b14-ab28-fe6f64e7b9c9 The web browser plugin with gpt 3.5